content/cli/orch_cli/apply.md

```diff
@@ -0,0 +1,238 @@
+---
+layout: bt_wiki
+title: apply
+category: Docs
+draft: false
+abstract: Cloudify's Command-Line Interface
+aliases: /cli/apply/
+---
+
```

Comment:
We should add some preface explaining the main usage scenario. something like:
When running frequent iterations of the same blueprint, for example during blueprint development and testing, it is useful to be able to quickly deploy and install the updated blueprint, overriding the existing deployment with the new changes. `cfy apply` is designed to assist the developer in such scenarios.